docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - FIDESOPS__LOG_PII=${FIDESOPS__LOG_PII}
       - FIDESOPS__HOT_RELOAD=${FIDESOPS__HOT_RELOAD}
       - FIDESOPS__ROOT_USER__ANALYTICS_ID=${FIDESOPS__ROOT_USER__ANALYTICS_ID}
+      - FIDESOPS__EXECUTION__WORKER_ENABLED=True
```

Comment:
Did you intentionally leave this in? This should only be set to `True` if the run command invokes the worker too.

Reply:
I didn't realize I added that. I'll remove.